jsdoc.json

```diff
@@ -5,7 +5,17 @@
   },
   "opts": {
     "recurse": true,
-    "destination": "../yeoman-generator-doc"
+    "destination": "../yeoman-generator-doc",
+    "template": "node_modules/tui-jsdoc-template"
+  },
+  "templates": {
+    "logo": {
+        "url": "https://cloud.githubusercontent.com/assets/441011/20215255/4c2a7246-a813-11e6-826e-80a78b30e9e5.png",
```

Comment:
Hej @eddiemonge,

the image I reference in this line is a copy from https://github.com/yeoman/yeoman.github.io/raw/source/app/assets/img/logo.png I uploaded to »the cloud«. 

Reason: I wanted to make sure the image won’t break when the Yeoman website changes.

Do you want me to reference the »original« image instead? This would lead to another PR since @SBoudrias already merged this one.